### DISCUSSION TITLE
[Feature] stacking edited photos / auto stacking based on filename

### DISCUSSION THREAD

**Discussion by xttlegendapi** [immich-app/immich#11707] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, I just wanted to brainstorm a bit about this. I like many others take pictures with a camera in a raw format, and then maybe edit it and export it as a jpg. And I also came across the scenario where you have normal videos and stabilized videos. normal ones are .mp4 and stabilized videos are .stab.mkv.
Stacking them all manually is a pain atm. 
So it would be great if there could be a better workflow for this.
 - Stacking based on hash or image data would not always work because of the small edits made
 - what could be an option is stacking based on filename/extention, if it has the same filename but another file extension and another check could be that it has the same metadata (eg time)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by schuhbacca** [comment]
https://immich.app/roadmap/

See roadmap, plan for autostacking is already on there.

**Reply by xttlegendapi** [reply]
But also in this way? Anywhere I can find details about this?

**Reply by Qhilm** [reply]
here are some more details: https://github.com/immich-app/immich/discussions/2479

I'm not sure manual stacking is something planned. But the deduplicate function is able to stack picture, so as long as it detects your two versions as duplicates, you should be able to stack them using this tool.
